### ISSUE TITLE
Servizi Anac esistenti - ESPD

### ISSUE THREAD

**Issue by ferdinandjmaddela** [anticorruzione/npa#18]
E' prevista l'implementazione in NPA dei servizi già esistenti riguardo l'ESPD (recupero criteri, creazione della richiesta espd, validazione) ? Quando saranno disponibile le specifiche?

**Comment by supportoAnac** [comment]
No, attualmente non è prevista l'esposizione di servizi specifici sull'ESPD. PCP prevede però tramite il servizio di /verifica-appalto la validazione dell'intera istanza dell'appalto, che di conseguenza valida anche l'ESPD se prevista dalla scheda.
